client/app/templates/json-api-errors.hbs

```diff
@@ -0,0 +1,9 @@
+{{#if errors}}
```

Comment:
Handlebars partials support passing in parameters as well: http://handlebarsjs.com/partials.html <http://handlebarsjs.com/partials.html>

> On Jan 10, 2016, at 3:03 PM, Matthew Philyaw <notifications@github.com> wrote:
> 
> In client/app/templates/json-api-errors.hbs <https://github.com/voluspa/yggdrasil/pull/14#discussion_r49280668>:
> 
> > @@ -0,0 +1,9 @@
> > +{{#if errors}}
> Oh yeah look at the login-form.hbs I actually render this as a partial. So it takes on the context of the template it's included in.
> 
> Figured this would be very common and don't know if it needs to a component just include it partially.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/voluspa/yggdrasil/pull/14/files#r49280668>.
>